watson-developer-cloud/discovery-components
# feat: remove pdfFallback component

#### What do these changes do/fix?

This removes the pdfFallback component, since this view relies heavily on text_mappings. Text_mappings has proven to be unreliable for document visualization.

#### How do you test/verify these changes?

Run `yarn test` and see that no tests fail.
Run `yarn storybook` and see that `PdfFallback` is no longer in the `DocumentPreview` components list.

ZenHub issue
https://github.ibm.com/Watson-Discovery/docviz-squad-issue-tracker/issues/229
